docs/ADVANCED-USAGE.md

```diff
@@ -63,3 +63,16 @@ $ npx convert ./browsers.html -f="./my-bookmarks.yaml"
 $ ls
 my-bookmarks.yaml
 ```
+
+## Check
+
+Check whether current generated files are up to date with the yaml file.
```

Comment:
I wonder if its worth documenting here why you might want to use this?

Reply:
Added some documentation, hope that's what you meant.